packages/components/src/mobile/bottom-sheet/link-suggestion-item-cell.native.js

```diff
@@ -36,7 +39,7 @@ const getSummaryForType = ( type ) => {
 };
 
 // we use some Cell styles here with a column flex-direction
-function LinkSuggestionItemCell( { suggestion, onLinkPicked } ) {
+function LinkSuggestionItemCell( { suggestion, onLinkPicked, ...props } ) {
```

Comment:
> I am not a big fan of this pattern {...props} mostly since it makes it harder to find the props that are being passed down. Do we need to pass down all the props? Why are we doing this in this case?

👆, @enejb

Reply:
I added this to my `TODO` list for a future PR.